src/studio/src/designer/frontend/packages/schema-editor/src/SchemaEditorApp.test.tsx

```diff
@@ -29,5 +29,5 @@ test('renders schema editor app', () => {
       />,
     );
   });
-  expect(wrapper.find('SchemaEditorApp')).toHaveLength(1);
+  expect(screen.getByTestId('schema-editor')).toBeDefined();
```

Comment:
A bit nitpick perhaps, but I really prefer using `toBeInTheDocument()`/`.not.toBeInTheDocument()` for these types of checks, as its very clear about the meaning - this selector should be found in the DOM. `toBeDefined()`/`toBeNull()`/`toHaveLenght(0)` doesnt really convey that same meaning. 

```suggestion
  expect(screen.getByTestId('schema-editor')).toBeInTheDocument();
```